code/game/machinery/Sleeper.dm

```diff
@@ -661,7 +665,17 @@
 						playsound(src, 'sound/effects/pop.ogg', 50, 1)
 						H.my_appearance.h_style = "Popped Hair"
 						H.update_hair()
-				else if(isjusthuman(H) && Holiday == APRIL_FOOLS_DAY)
+				else if(isjusthuman(H) && (Holiday == APRIL_FOOLS_DAY || galize == 1))
```

Comment:
just galize is fine, since it's a binary var

Reply:
if you make the holiday check from before turn the galize var on, you could remove this holiday check